test/utils/image/manifest.go

```diff
@@ -188,6 +188,8 @@ const (
 	NFSProvisioner
 	// Nginx image
 	Nginx
+	// NginxCopy image
```

Comment:
can you please extend the comment to make sure there it's clear we want the same SHA here, but different tag